code/modules/mob/living/simple_animal/constructs.dm

```diff
@@ -306,6 +307,8 @@
 		if(!LAZYLEN(parts))
 			if(undismembermerable_limbs) //they have limbs we can't remove, and no parts we can, attack!
 				return ..()
+			C.Weaken(30)
+			visible_message("<span class='danger'>[src] feints a deathblow against [C], they freeze in terror!</span>")

```

Comment:
you don't really need this line honestly, and I think it looks weird
most players with no limbs won't even notice they're stunned